src/utils/session.ts

```diff
@@ -14,25 +22,15 @@ effect(() => {
   }
 });
 
-export function createRoom() {
-  if (!roomName.value) {
-    alert("A room name is required.");
-    return;
-  }
-  if (!playerName.value) {
-    alert("A player name is required.");
-    return;
-  }
-  peer.value?.destroy();
-  peer.value = new Peer(hostId.value, PEER_JS_OPTIONS);
-  console.info(
-    `Create a room with name ${roomName} as ${playerName} and wait for players to join.`
-  );
-  playerMap.value = new Map([[peer.value.id, playerName.value]]);
-}
+effect(() => {
+  if (!playerMap.value) return;
+  unreadPlayerListChanges.value = true;
```

Comment:
Can you help me understand the logic within this `effect`?

Reply:
This whole effect is for `playerMap` changes.

The `unreadPlayerListChanges` works similarly to `unreadChatMessages`. If the `playerMap` is ever changed, the effect cause `unreadPlayerListChanges` to be set as true. Then, it would be reset by the effect function in the player list page. 

If you let a new player joins and navigate the other players to pages other than the player list (similar to the chat page), it shows an indicator that you have unchecked player changes.